File: ncDados/dVenda.vb
Namespace nsVenda

    Public Class ColecaoVenda
        Inherits List(Of dVenda)
    End Class

    Public Class dVenda

        Private _controle As Integer
        Private _usuarioId As Integer
        Private _clienteId As Integer
        Private _ordemServicoId As String
        Private _data As DateTime
        Private _dataFim As DateTime
        Private _dinheiro As Decimal
        Private _cheque As Decimal
        Private _chequePre As Decimal
        Private _cartaoDebito As Decimal
        Private _cartaoCredito As Decimal
        Private _crediario As Decimal
        Private _crediarioPagamento As Decimal
        Private _parcelas As Integer
        Private _desconto As Decimal
        Private _condicao As Integer
        Private _recebido As Decimal
        Private _troco As Decimal
        Private _total As Decimal
        Private _troca As Decimal
        Private _vale As Decimal
        Private _defeito As Decimal
        Private _retirada As Decimal
        Private _terminal As String
        Private _valeEmitido As Decimal
        Private _vendedor As String
        Private _caixa As String
        Private _valorProduto As Nullable(Of Decimal)
        Private _valorCusto As Nullable(Of Decimal)

        Public Property controle() As Integer
            Get
                Return _controle
            End Get
            Set(ByVal value As Integer)
                _controle = value
            End Set
        End Property

        Public Property usuarioId() As Integer
            Get
                Return _usuarioId
            End Get
            Set(ByVal value As Integer)
                _usuarioId = value
            End Set
        End Property

        Public Property clienteId() As Integer
            Get
                Return _clienteId
            End Get
            Set(ByVal value As Integer)
                _clienteId = value
            End Set
        End Property

        Public Property ordemServicoId() As String
            Get
                Return _ordemServicoId
            End Get
            Set(ByVal value As String)
                _ordemServicoId = value
            End Set
        End Property

        Public Property Data() As DateTime
            Get
                Return _data
            End Get
            Set(ByVal value As DateTime)
                _data = value
            End Set
        End Property

        Public Property DataFim() As DateTime
            Get
                Return _dataFim
            End Get
            Set(ByVal value As DateTime)
                _dataFim = value
            End Set
        End Property

        Public Property Dinheiro() As Decimal
            Get
                Return _dinheiro
            End Get
            Set(ByVal value As Decimal)
                _dinheiro = value
            End Set
        End Property

        Public Property Cheque() As Decimal
            Get
                Return _cheque
            End Get
            Set(ByVal value As Decimal)
                _cheque = value
            End Set
        End Property

        Public Property ChequePre() As Decimal
            Get
                Return _chequePre
            End Get
            Set(ByVal value As Decimal)
                _chequePre = value
            End Set
        End Property

        Public Property CartaoDebito() As Decimal
            Get
                Return _cartaoDebito
            End Get
            Set(ByVal value As Decimal)
                _cartaoDebito = value
            End Set
        End Property

        Public Property CartaoCredito() As Decimal
            Get
                Return _cartaoCredito
            End Get
            Set(ByVal value As Decimal)
                _cartaoCredito = value
            End Set
        End Property

        Public Property Crediario() As Decimal
            Get
                Return _crediario
            End Get
            Set(ByVal value As Decimal)
                _crediario = value
            End Set
        End Property

        Public Property CrediarioPagamento() As Decimal
            Get
                Return _crediarioPagamento
            End Get
            Set(ByVal value As Decimal)
                _crediarioPagamento = value
            End Set
        End Property

        Public Property Parcelas() As Integer
            Get
                Return _parcelas
            End Get
            Set(ByVal value As Integer)
                _parcelas = value
            End Set
        End Property

        Public Property Desconto() As Decimal
            Get
                Return _desconto
            End Get
            Set(ByVal value As Decimal)
                _desconto = value
            End Set
        End Property

        Public Property Condicao() As Integer
            Get
                Return _condicao
            End Get
            Set(ByVal value As Integer)
                _condicao = value
            End Set
        End Property

        Public Property Recebido() As Decimal
            Get
                Return _recebido
            End Get
            Set(ByVal value As Decimal)
                _recebido = value
            End Set
        End Property

        Public Property Troco() As Decimal
            Get
                Return _troco
            End Get
            Set(ByVal value As Decimal)
                _troco = value
            End Set
        End Property

        Public Property Total() As Decimal
            Get
                Return _total
            End Get
            Set(ByVal value As Decimal)
                _total = value
            End Set
        End Property

        Public Property Troca() As Decimal
            Get
                Return _troca
            End Get
            Set(ByVal value As Decimal)
                _troca = value
            End Set
        End Property

        Public Property Vale() As Decimal
            Get
                Return _vale
            End Get
            Set(ByVal value As Decimal)
                _vale = value
            End Set
        End Property

        Public Property ValeEmitido() As Decimal
            Get
                Return _valeEmitido
            End Get
            Set(ByVal value As Decimal)
                _valeEmitido = value
            End Set
        End Property

        Public Property Defeito() As Decimal
            Get
                Return _defeito
            End Get
            Set(ByVal value As Decimal)
                _defeito = value
            End Set
        End Property

        Public Property Retirada() As Decimal
            Get
                Return _retirada
            End Get
            Set(ByVal value As Decimal)
                _retirada = value
            End Set
        End Property

        Public Property Terminal() As String
            Get
                Return _terminal
            End Get
            Set(ByVal value As String)
                _terminal = value
            End Set
        End Property

        Public Property Vendedor() As String
            Get
                Return _vendedor
            End Get
            Set(ByVal value As String)
                _vendedor = value
            End Set
        End Property

        Public Property Caixa() As String
            Get
                Return _caixa
            End Get
            Set(ByVal value As String)
                _caixa = value
            End Set
        End Property

        Public Property valorProduto() As Nullable(Of Decimal)
            Get
                Return _valorProduto
            End Get
            Set(ByVal value As Nullable(Of Decimal))
                _valorProduto = value
            End Set
        End Property

        Public Property valorCusto() As Nullable(Of Decimal)
            Get
                Return _valorCusto
            End Get
            Set(ByVal value As Nullable(Of Decimal))
                _valorCusto = value
            End Set
        End Property

    End Class

End Namespace

File: ncModelos/dVenda.cs
using System;
using System.Collections.Generic;

namespace nsVenda
{
    public class ColecaoVenda : List<dVenda>
    {
    }

    public class dVenda
    {
        private int _controle;
        private int _usuarioId;
        private int _clienteId;
        private string _ordemServicoId;
        private DateTime _data;
        private DateTime _dataFim;
        private decimal _dinheiro;
        private decimal _pix;
        private decimal _cheque;
        private decimal _chequePre;
        private decimal _cartaoDebito;
        private decimal _cartaoCredito;
        private decimal _crediario;
        private decimal _crediarioPagamento;
        private int _parcelas;
        private decimal _desconto;
        private int _condicao;
        private decimal _recebido;
        private decimal _troco;
        private decimal _total;
        private decimal _troca;
        private decimal _vale;
        private decimal _defeito;
        private decimal _retirada;
        private string _terminal;
        private decimal _valeEmitido;
        private string _vendedor;
        private string _caixa;
        private string _txID;
        private decimal? _valorProduto;
        private decimal? _valorCusto;
        private decimal _original;
        private string _chave;

        public int controle
        {
            get => _controle;
            set => _controle = value;
        }

        public int usuarioId
        {
            get => _usuarioId;
            set => _usuarioId = value;
        }

        public int clienteId
        {
            get => _clienteId;
            set => _clienteId = value;
        }

        public string ordemServicoId
        {
            get => _ordemServicoId;
            set => _ordemServicoId = value;
        }

        public DateTime Data
        {
            get => _data;
            set => _data = value;
        }

        public DateTime DataFim
        {
            get => _dataFim;
            set => _dataFim = value;
        }

        public decimal Dinheiro
        {
            get => _dinheiro;
            set => _dinheiro = value;
        }

        public decimal Pix
        {
            get => _pix;
            set => _pix = value;
        }

        public decimal Cheque
        {
            get => _cheque;
            set => _cheque = value;
        }

        public decimal ChequePre
        {
            get => _chequePre;
            set => _chequePre = value;
        }

        public decimal CartaoDebito
        {
            get => _cartaoDebito;
            set => _cartaoDebito = value;
        }

        public decimal CartaoCredito
        {
            get => _cartaoCredito;
            set => _cartaoCredito = value;
        }

        public decimal Crediario
        {
            get => _crediario;
            set => _crediario = value;
        }

        public decimal CrediarioPagamento
        {
            get => _crediarioPagamento;
            set => _crediarioPagamento = value;
        }

        public int Parcelas
        {
            get => _parcelas;
            set => _parcelas = value;
        }

        public decimal Desconto
        {
            get => _desconto;
            set => _desconto = value;
        }

        public int Condicao
        {
            get => _condicao;
            set => _condicao = value;
        }

        public decimal Recebido
        {
            get => _recebido;
            set => _recebido = value;
        }

        public decimal Troco
        {
            get => _troco;
            set => _troco = value;
        }

        public decimal Total
        {
            get => _total;
            set => _total = value;
        }

        public decimal Troca
        {
            get => _troca;
            set => _troca = value;
        }

        public decimal Vale
        {
            get => _vale;
            set => _vale = value;
        }

        public decimal ValeEmitido
        {
            get => _valeEmitido;
            set => _valeEmitido = value;
        }

        public decimal Defeito
        {
            get => _defeito;
            set => _defeito = value;
        }

        public decimal Retirada
        {
            get => _retirada;
            set => _retirada = value;
        }

        public string Terminal
        {
            get => _terminal;
            set => _terminal = value;
        }

        public string Vendedor
        {
            get => _vendedor;
            set => _vendedor = value;
        }

        public string Caixa
        {
            get => _caixa;
            set => _caixa = value;
        }

        public string TXID
        {
            get => _txID;
            set => _txID = value;
        }

        public decimal? valorProduto
        {
            get => _valorProduto;
            set => _valorProduto = value;
        }

        public decimal? valorCusto
        {
            get => _valorCusto;
            set => _valorCusto = value;
        }

        public decimal valorOriginal
        {
            get => _original;
            set => _original = value;
        }

        public string Chave
        {
            get => _chave;
            set => _chave = value;
        }
    }
}
